src/csharp/Examples/WPF/Microsoft.Azure.Kinect.Sensor.Examples.WPFViewer.csproj

```diff
@@ -159,8 +159,8 @@
       <Link>k4a.pdb</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="$(K4aBinaryDirectory)\depthengine_2_0.dll">
-      <Link>depthengine_2_0.dll</Link>
+    <Content Include="$(K4aBinaryDirectory)\depthengine_*.dll">
+      <Link>%(Filename)%(Extension)</Link>
```

Comment:
@wes-b This allows us to pick up whatever depth engine you have on your system, however if you sync a change to the depth engine while you have the solution open, it won't pick up the changes in files. This could be a little brittle. Thoughts?

Reply:
Reverting to explicit links so that it is clear when using Visual Studio what version is required.